src/main/java/com/gromoks/filestatistic/Run.java

```diff
@@ -0,0 +1,45 @@
+package com.gromoks.filestatistic;
+
+import com.gromoks.filestatistic.dao.config.JdbcConnection;
+import com.gromoks.filestatistic.dao.io.InputData;
+import com.gromoks.filestatistic.service.FileProcessing;
+import com.gromoks.filestatistic.dao.jdbc.JdbcStatisticDao;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Run {
+
+    private static FileProcessing fileProcessing;
+
+    public static void main(String[] args) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+            System.out.println("Please enter file path:");
+            Path path = Paths.get(bufferedReader.readLine());
+
+            if (Files.exists(path)) {
+                init();
+                String wordDelimiter = "\\s+";
+                fileProcessing.process(path, wordDelimiter);
```

Comment:
Better to move this value to getLineStatistic(), because it is constant specific only for this method. If you want to define this value as parameter then would be better to store it in *.parameters file